conf/bookkeeper.conf

```diff
@@ -642,10 +642,10 @@ diskCheckInterval=10000
 ## Metadata Service settings
 #############################################################################
 
-# metadata service uri that bookkeeper is used for loading corresponding metadata driver and resolving its metadata service location
+# Metadata service uri that bookkeeper uses for loading the corresponding metadata driver and resolving its metadata service location
 # Examples: 
-#  - metadataServiceUri=metadata-store:zk:my-zk-1:2181
-#  - metadataServiceUri=metadata-store:etcd:http://my-etcd:2379
+#  - metadataServiceUri=zk://my-zk-1:2181/ledgers
+#  - metadataServiceUri=etcd:http://my-etcd:2379
```

Comment:
I'm not quite sure whether this config pattern is correct.

I'm using the `metadata-store:` variant in prod env.

cc @merlimat @eolivelli